### PR TITLE
gnome: add build option gir

### PR DESCRIPTION
**Pull request by pullmoll**
Make some more packages possibly cross compilable.

### CODE REVIEW

**Comment by xtraeme** [comment]
:+1:

**Comment by xtraeme** [comment]
btw @pullmoll if you are a mame dev feel free to take maintainership of the mame pkg.

**Comment by pullmoll** [comment]
Yes, I can do that, if you like, with the next update.

**Comment by xtraeme** [comment]
Sure, thanks!

**Comment by xtraeme** [comment]
Got 98% gnome built with musl locally.

**Comment by pullmoll** [comment]
:pray: :grin:

**Comment by xtraeme** [comment]
Hmm sorry, committed a change to mousetweaks... please rebase.

**Comment by xtraeme** [comment]
index: added `gnome-3.16.0_4' (x86_64-musl).
index: 149 packages registered.

**Comment by pullmoll** [comment]
Oops, I think I dropped your change to moustweaks now?

File: srcpkgs/mousetweaks/template [hunk 1]
```diff
@@ -3,7 +3,7 @@ pkgname=mousetweaks
 version=3.12.0
 revision=2
```
Comment (xtraeme) [inline srcpkgs/mousetweaks/template]: Please revbump this one, otherwise won't be built.

**Comment by xtraeme** [comment]
:+1:

**Pull request merged**